### PR TITLE
Clarified 4.1.3

### PR DESCRIPTION
**Pull request by jfhector**

### CODE REVIEW

**Comment by taliazdc** [comment]
Sorry, again just leaving comments everywhere! Let me know if you would prefer it elsewhere. This one reads a bit wordy with passive voice. 

>Identify status messages (like error or success notifications) in code so assistive technologies can communicate them without requiring users to change positions on the page. 

There's probably a reason why we are starting these with "ensure... make sure..." which i'm missing though!

**Comment by jfhector** [comment]
Thanks for the review.

I use 'ensure' and 'make sure' a lot, because each success criterion is an outcome. i.e. you can test whether that 'outcome' is the case or not.

That's how they work in the WCAG:
* success criteria (e.g. 4.1.3) are testable assertion (what I call an outcome here)
  * e.g. 4.1.3: "Status Messages: In content implemented using markup languages, status messages can be programmatically determined through role or properties such that they can be presented to the user by assistive technologies without receiving focus."

* the guidelines that sit above the success criteria are not designed to be testable (i.e. they are not phrased like assertions)
  * e.g. 4.1: "Compatible: Maximize compatibility with current and future user agents, including assistive technologies."

In practice, this means that there are two ways to phrase the success criteria (that I could think of):
* Option A is to use language like 'ensure' or 'make sure' a lot (this is what the GDS' WCAG primer does, and I thought that it make things feel much more like a concrete to-do list of things to do)
* Option B (the alternative) would be to use language like 'must' a lot. (Like in the Deque checklist), or to make these 'must' explicit by just describing the assertion (like the WCAG)

So far I've picked Option A, because I liked how the GDS WCAG primer worked, and I feel that it's not perfect but I like it better than approach B.

Whichever approach we take, we need to be consistent across all the success criteria.

What do you think?

**Comment by jfhector** [comment]
My previous comment was to try to explain the approach I've been following.

Concerning your specific point: yes, for sure. Let's try to simplify.

#### Intent of the PR

The addition I made with this pull request, was to clarify that it's more than just about "so that assistive technologies can convey them to users.".

It's also about making sure that assistive technologies can do that without moving the focus away from where it was, and disturbing the user's place on a page.

I think that that is important to specify. I'm just trying to say it simply and clearly.

#### Another attempt

Here's another attempt:

> Make sure that status messages (like error, success or 'system busy' notifications) are identified in code so that screen readers can communicate them without disturbing the user’s place in a page.

What do you think?

File: README.md [hunk 1]
```diff
@@ -168,3 +168,5 @@ Your website/app must work with different web browsers and/or assistive technolo
 * [Name, Role and State of interactive controls](): Make sure the code of each page enables assistive technologies to discover the purpose of every feature, the way that feature is identified, and the state it is currently in.
 
 * [Status messages](): Make sure status messages are identified in code, so that assistive technologies can convey them to users.
+
+* [Status messages](): Make sure status messages (like error or success notifications) are identified in code in such a way that assistive technologies can communicate them without requiring users to change their position on the page.
```
Comment (KaneCheshire) [inline README.md:170]: I think you need to delete line 170 now?

**Review by KaneCheshire — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jfhector) [reply README.md:170]: Yes, well spotted

**Review by KaneCheshire — APPROVED** [review]

**Comment by KaneCheshire** [comment]
What's happening with this PR @jfhector? Should we just close it or is it still valid?

**Comment by jfhector** [comment]
I intend to close this pull request soon, when I get to completing the 4.1.3 detail page. When I do that, I'll make up my mind about the best wording for the summary, and I'll update both the index.html file and the detailed guideline file.

In the meantime, do share your point of views on the best wording for SC 4.1.3

**Comment by jfhector** [comment]
Closing this pull request because I've opened another pull request (https://github.com/theappbusiness/accessibility-guidelines/pull/76) which covers the same change, taking @taliazdc feedback into account.
